keyboards/bm60poker/keymaps/ipetepete/keymap.c

```diff
@@ -0,0 +1,132 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+_qwerty,
+_gamer,
+_colemak,
+_fn,
+_rgb,
+_spcfn
+};
+
+
+enum keycodes {
+glight = SAFE_RANGE,
+gamer,
```

Comment:
You aren't using the `glight` keycode, so it can be removed:

```suggestion
gamer = SAFE_RANGE,
```